addons/forum-post-countdown/userscript.js

```diff
@@ -0,0 +1,38 @@
+export default async function ({ addon, msg }) {
+  const countdown = localStorage.getItem("sa-forum-post-countdown");
+  if (!countdown) {
+    return;
```

Comment:
I was wrong about my review because now this script will never run since the storage item is set later.

Reply:
Should now be fixed (among with a bunch of other things)